grafana-plugin/src/state/rootBaseStore/index.ts

```diff
@@ -56,6 +56,9 @@ export class RootBaseStore {
   @observable
   initializationError = null;
 
+  @observable
+  currentlyUndergoingMaintenance = false;
```

Comment:
can we reuse `initializationError` instead of adding new property `currentlyUndergoingMaintenance` here?

Reply:
I wanted to add this new attribute to distinguish between a setup error and maintenance mode. For maintenance mode, I think we should disable the "Retry" and "Configure Plugin" buttons ([here](https://github.com/grafana/oncall/pull/1917/files#diff-e3115759934c8107978a15e1ab02872bba90969b069b6c0d3dadec4a83c1b119R50-R61))